src/solvers/dgmulti/types.jl

```diff
@@ -66,6 +66,17 @@ function DGMulti(; polydeg=nothing,
           polydeg=polydeg, kwargs...)
 end
 
+# dispatchable constructor for DGMulti to allow for specialization
+function DGMulti(element_type::AbstractElemShape, 
+  approximation_type::TensorProductWedge, 
```

Comment:
This type is defined in terms of RefElemData - this may be slightly confusing since RefElemData is outside of Trixi.jl. 

What do you think about changing the constructor so that we introduce either 
- `solver` variables for both the triangle and the line, or
- solver parameters for both the triangle and line?

Reply:
What about passing a Tuple as ```polydeg```?